lib/regex.js

```diff
@@ -18,7 +18,7 @@ function getNotesRegex(noteKeywords) {
     return reNomatch;
   }
 
-  return new RegExp('^\\s*(' + join(noteKeywords, '|') + ')[:\\s]+(.*)', 'i');
+  return new RegExp('^[\\s|\\W]*(' + join(noteKeywords, '|') + ')[:\\s]+(.*)', 'i');
```

Comment:
See https://github.com/conventional-changelog/conventional-commits-parser/commit/6e13789a0989fc898efdaa601e781ec98a02cd89,
there was a bug. I'm wondering if you could make this regex a bit more strict? Just in case it breaks something else.